packages/ui/src/views/chatflows/APICodeDialog.js

```diff
@@ -347,9 +356,12 @@ async function query(formData) {
     const response = await fetch(
         "${baseURL}/api/v1/prediction/${dialogProps.chatflowid}",
         {
-            headers: { Authorization: "Bearer ${selectedApiKey?.apiKey}" },
+            headers: {
+                Authorization: "Bearer ${selectedApiKey?.apiKey}",
+                "Content-Type": "application/x-www-form-urlencoded"
+            },
             method: "POST",
-            body: formData
+            body: new URLSearchParams(formData)
```

Comment:
do we need `new URLSearchParams` ? 

actually i dont think we should set `"Content-Type": "application/x-www-form-urlencoded"` according to many [answers](https://stackoverflow.com/questions/62644988/using-formdata-inside-fetch-api-is-not-working) online, just let the fetch decide for you

Reply:
I see.

then I'll revert to the changes related to the form.